meeteval/wer/wer/time_constrained.py

```diff
@@ -239,10 +248,13 @@ def _repr_pretty_(self, p, cycle):
 TimeMarkedTranscriptLike = 'TimeMarkedTranscript | STM | List[Segment]'
 
 
+def apply_collar(s: TimeMarkedTranscript, collar: float):
+    return replace(s, timings=[(max(t[0] - collar, 0), t[1] + collar) for t in s.timings])
```

Comment:
Is negative time not allowed?

Reply:
You are right, negative time is currently allowed. I copied this to keep the old behavior. An old implementation used unsigned types. I'll remove the max operation

Reply:
Done